clients/imodelhub/src/imodelhub/Events.ts

```diff
@@ -221,7 +222,7 @@ export class VersionEvent extends IModelHubEvent {
 }
 
 /** Sent when a new [[Checkpoint]] is generated. [[Checkpoint]]s can be generated daily when there are new [[ChangeSet]]s pushed or when a new [[Version]] is created.
- * @beta
+ * @internal

```

Comment:
I can see some value to making this public, but I like the conservative approach - we can always make it public if someone requests this.